utils/cloudinary.ts

```diff
@@ -0,0 +1,10 @@
+import { v2 as cloudinary } from 'cloudinary';
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
```

Comment:
configure these environment variables in `environment.d.ts` as well for ts support...?